tools/generators/dependency-mismatch/index.spec.ts

```diff
@@ -65,28 +69,30 @@ describe('dependency-mismatch generator', () => {
     `);
   });
 
-  it('should ignore non-converged dependency', async () => {
```

Comment:
what happened to test case for ignoring non converged deps ?

Reply:
because we don't ignore non-converged deps anymore ?

Reply:
gotcha, thx